ui/src/utils/tableUtils.ts

```diff
@@ -5,14 +5,14 @@ export const getSortParams = (
   activeSortIndex: number | null,
   activeSortDirection: 'asc' | 'desc',
   setActiveSortIndex: React.Dispatch<React.SetStateAction<number | null>>,
-  setActiveSortDirection: React.Dispatch<React.SetStateAction<'asc' | 'desc'>>,
+  setActiveSortDirection: React.Dispatch<React.SetStateAction<'asc' | 'desc'>>
```

Comment:
Lots of changes in this pr include the `,` after last entry in the list, lets add it back for consistency

Reply:
Lint didnt like it:

```
./src/utils/tableUtils.ts
8:79  Error: Delete `,`  prettier/prettier
```